src/test/java/com/thebois/models/beings/ColonyTests.java

```diff
@@ -49,4 +61,78 @@ public void updateCallsUpdateOnAllBeings() {
         }
     }
 
+    @ParameterizedTest
+    @MethodSource("getItemTypes")
+    public void emptyColonyInventoryIsEmpty(final ItemType itemType) {
```

Comment:
Didn't notice this at first, but you've basically just copied all the tests for inventory into `ColonyTests`, haven't you? That really smells.

A single behaviour shouldn't be tested in exactly the same way twice in two different places, what you actually want to test here *that Colony delegates to Inventory correctly*. 

To do this, you can use a mocked `Inventory`, pass it in to `Colony`, call the delegating method you want to test, and then verify by calling `verify(mockInventory, times(1)).addItem(any())`.

Reply:
Now, since this inventory inside colony will be removed in an upcoming Expansion, it may not be worth it to spend the time to fix this.

Reply:
Agree and agree